templates/service-template/docker-compose.yml

```diff
@@ -30,21 +31,21 @@ services:
       start_period: 10s # Time to wait before starting health checks
     restart: always
 
-  # SERVICE
-  SERVICE:
-    image: lscr.io/linuxserver/SERVICE:latest # Image to be used
-    network_mode: service:tailscale-SERVICE # Sidecar configuration to route SERVICE through Tailscale
```

Comment:
This only works when `service:` is present. Please alter the `network_mode: tailscale` to `network_mode: service:tailscale`